setup.py

```diff
@@ -7,12 +7,13 @@
 setuptools.setup(
     name="pello",
     version="1.0.0",
-    author="",
-    author_email="",
-    description="A package that makes it easy to create Pypi packages",
+    author="Tomáš Hrnčiar",
+    author_email="tomas.hrnciar@me.com",
+    description="An example Python Hello World package",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    url="",
+    url="https://github.com/hrnciar/pello",
+    license="MIT",
```

Comment:
Examples like this are best under some very very very free license, such as CC0, so people can copy paste from them without worrying about where to put attribution.